transformer/node_dispatcher.ts

```diff
@@ -41,55 +32,53 @@ const isMethodCall = (node: ts.Node, method: string): boolean => {
     }
 };
 
-// Determine if given node is RxJS Pipe Statement.
-const isPipeStatement = (node: ts.Node): boolean => isMethodCall(node, 'pipe');
+// Check if node is pipe property access expression.
+const isPipePropertyAccessExpr = (node: ts.Node): boolean => {
+    if (ts.isCallExpression(node) && ts.isPropertyAccessExpression(node.expression)) {
+        return node.expression.name.getText() === 'pipe' ? true : false;
+    }
+    return false;
+};
 
 // Determine if given node is RxJS Subscribe Statement.
 const isSubscribeStatement = (node: ts.Node): boolean => isMethodCall(node, 'subscribe');
 
-// Wrap operator into wrapOperator.
-const wrap = (operator: string): string => {
-    const capitalized = operator[0].toUpperCase() + operator.slice(1);
-    return `wrap${capitalized}`;
-};
-
-// Classify given node, return node classification and import statement.
-const classify = (node: ts.Node): [NodeType, string | null] => {
+// Classify given node, return node classification.
+const classify = (node: ts.Node): NodeType => {
     let classification: NodeType = 'UNCLASSIFIED';
-    let importStatement = null;
 
-    const [foundCreationOperator, creationOperator] = isRxJSCreationOperator(node);
-    if (foundCreationOperator) {
+    if (isRxJSCreationOperator(node)) {
         classification = 'RXJS_CREATION_OPERATOR';
-        importStatement = wrap(creationOperator);
     }
 
-    isPipeStatement(node) && (classification = 'RXJS_PIPE_OPERATOR');
-    isSubscribeStatement(node) && (classification = 'RXJS_SUBSCRIBE');
+    if (isPipePropertyAccessExpr(node)) {
```

Comment:
are these if statements exclusive? Or is the execution order relevant?
If they are exclusive please use `else if ` to express this